### ISSUE TITLE
.kbb read fails on long lines without throwing error

### ISSUE THREAD

**Issue by ashtonomy** [VisualText/vscode-nlp#881]
I'm running into an issue with reading .kbb files with attributes of containing arrays of string values beyond a certain length. Here is a minimum example .kbb file, [test_kbb.txt](https://github.com/VisualText/vscode-nlp/files/12553938/test_kbb.txt), which fails to be read in correctly. Here's an outline:
```
parent
    a
        attr_1:["a00.01","a00.01,"a00.01"]
    b
        attr_2:["a00.01",...,"a00.01"] 
        attr_3:["a00.01","a00.01","a00.01"]
    c
        attr_4:["a00.01","a00.01,"a00.01"]
```
Where attr_2 contains ~240 values . If you call `DisplayKB(findconcept(findroot(), "parent"), 2)` on this kb, the output will look like this:
```
parent
    a
        attr_1:["a00.01","a00.01,"a00.01"]
    b
```
In other words, the KB read fails on lines that are too long (by number of tokens, maybe. Changing "a00.01" to "a" works), but no error is thrown, and the analyzer continues to run, which can lead to hard to debug errors. 

Tested with engine version 2.10.2 and VisualText version 2.33.2
